test/test_data/gcc.py

```diff
@@ -0,0 +1,18 @@
+mapping_test_data = [
+    {
+        "module": "gcc",
```

Comment:
I think the problem is here: it's expecting `"product": "gcc"` instead of `module` -- did we forget to update the documentation somewhere and it still says module?

Reply:
Hey @terriko, I was referring to the test/README.md file for writing this, over there it is mentioned `"module"` instead of `"product"`.